src/api/changePassword.js

```diff
@@ -0,0 +1,48 @@
+const router = require("express").Router();
+const nodemailer = require("nodemailer");
+const cors = require("cors");
+const User = require("../models/User");
+const bodyParser = require("body-parser");
+const jwt = require("jsonwebtoken");
+const exjwt = require("express-jwt");
+const sendResponse = require("../utils/sendResponse");
```

Comment:
```
const {sendResponse} = require("../utils/sendResponse");
```

Reply:
Since you want to just the the correction function for that file

Reply:
You're right! Good catch.